src/qt/bitcoin.cpp

```diff
@@ -453,15 +464,15 @@ int GuiMain(int argc, char* argv[])
     QCoreApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
 #endif
 
-    BitcoinApplication app(*node);
+    BitcoinApplication app;
 
     /// 2. Parse command-line options. We do this after qt in order to show an error if there are problems parsing these
     // Command-line options take precedence:
-    node->setupServerArgs();
+    SetupServerArgs(node_context);
     SetupUIArgs(gArgs);
     std::string error;
-    if (!node->parseParameters(argc, argv, error)) {
-        node->initError(strprintf(Untranslated("Error parsing command line arguments: %s\n"), error));
+    if (!gArgs.ParseParameters(argc, argv, error)) {
```

Comment:
nit: (if feedback is addressed)

Could use 
```suggestion
    if (!args.ParseParameters(argc, argv, error)) {
```

here?

Reply:
re: https://github.com/bitcoin-core/gui/pull/35#discussion_r477205453

> Could use

(same feedback)